x-pack/packages/security/api_key_management/src/components/token_field.tsx

```diff
@@ -29,12 +28,25 @@ export interface TokenFieldProps extends Omit<EuiFieldTextProps, 'append'> {
   value: string;
 }
 
-export const TokenField: FunctionComponent<TokenFieldProps> = (props) => {
+export const TokenField: FunctionComponent<TokenFieldProps> = ({ value, ...props }) => {
   return (
-    <EuiFormControlLayout
+    <EuiFieldText
+      data-test-subj="apiKeyTokenField"
+      controlOnly
```

Comment:
Shoot, I left in this `controlOnly` prop by accident. Thanks so much for catching that Jeramy! 🙇 
```suggestion
```